admin/lib/validator/common/DateValidationData.js

```diff
@@ -0,0 +1,56 @@
+let DateValidationData = class {
+  day (day) {
```

Comment:
It would be nice to have JSDoc here detailing what params are expected here... strings, ints, HTML element Ids

Reply:
@jon-shipley Added